tests/test_grid.py

```diff
@@ -26,3 +26,14 @@ def test_get_grid_scrip():
     ds_test = pop_tools.get_grid('POP_gx3v7', scrip=True)
     ds_ref = xr.open_dataset(DATASETS.fetch('POP_gx3v7.nc'))
     assert ds_compare(ds_test, ds_ref, assertion='allclose', rtol=1e-14, atol=1e-14)
+
+
+def test_get_grid_to_netcdf():
+    for grid in pop_tools.grid_defs.keys():
+        print('-' * 80)
+        print(grid)
+        ds = pop_tools.get_grid(grid)
+        for format in ['NETCDF4', 'NETCDF3_64BIT']:
+            gridfile = f'{grid}_{format}.nc'
+            ds.to_netcdf(gridfile)
```

Comment:
Also, do we want to confirm that we are getting an identical dataset?

```suggestion
            ds.to_netcdf(gridfile, format=format)
            ds_2 = xr.open_dataset(gridfile)
            xr.testing.assert_identical(ds, ds_2)
```

Reply:
I think that's a good idea.  Currently, the test (as @mnlevy1981 created it) confirms that *writing* the dataset to file does not fail, which is the point.  Rereading the dataset and testing that it is identical seems (to me) to test something else...that `xarray` can round-trip via NetCDF.  I'm not sure that is necessary.